### PR TITLE
New/layout visualizer tests

### PR DESCRIPTION
**Pull request by GregoBalu**
I believe the last time the only thing it needed was the gitignore file? I don't know, I added that. Other than that nothing changed.

### CODE REVIEW

**Comment by kovacsgabor** [comment]
ok to test

**Comment by kovacsgabor** [comment]
restest this please

**Comment by kovacsgabor** [comment]
@GregoBalu, the tests fail because the layout.visualizer.tests project cannot be compiled for some reasons.

**Comment by GregoBalu** [comment]
It did work fine for me. It also works fine for me now. Did Jenkins die again probably?

**Comment by kovacsgabor** [comment]
retest this please

**Comment by kovacsgabor** [comment]
@gerazo, could you take a look at this test failure? I cannot understand what is the problem of Jenkins here.

File: dev/tests/hu.elte.txtuml.layout.visualizer.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: txtUML Layout Visualizer Tests
+Bundle-SymbolicName: hu.elte.txtuml.layout.visualizer.tests
+Bundle-Version: 0.3.0.qualifier
```
Comment (gerazo) [inline dev/tests/hu.elte.txtuml.layout.visualizer.tests/META-INF/MANIFEST.MF]: This should be 0.4.0

**Comment by gerazo** [comment]
This is the key sentence here (only visible at the end of maven build when you check the console output): Unqualified OSGi version 0.3.0.qualifier must match unqualified Maven version 0.4.0-SNAPSHOT

**Comment by kovacsgabor** [comment]
retest this please

**Pull request merged**